### PR TITLE
Make Port 443 configurable

### PR DESCRIPTION
**Pull request by ddragosd**
Sometimes the local ports `80` or `443` can't be allocated when starting docker-compose. This PR allows developers to configure a different port instead.

### Changelist:
- [x] Port `443` is configured via `OPENWHISK_GW_PORT`
- [x] Removed unnecessary port `80` 
- [x] Fixed the issue where controller logs wouldn't show up in `~/tmp/openwhisk/controller/logs/`
- [x] Updated documentation
- [x] Documented how to debug `controller` or `invoker` locally

### CODE REVIEW

File: docker-compose/Makefile [hunk 1]
```diff
@@ -221,16 +225,13 @@ setup:
 	mkdir -p ~/tmp/openwhisk/minio
 	cp -r ./apigateway/* ~/tmp/openwhisk/api-gateway-config/api-gateway/
 	cp -r ./apigateway/rclone ~/tmp/openwhisk
-       
  
-	$(shell cat ~/tmp/openwhisk/local.env) docker-compose --project-name openwhisk up minio 2>&1 > ~/tmp/openwhisk/setup.log &
```
Comment (ddragosd) [inline docker-compose/Makefile]: `cat ~/tmp/openwhisk/local.env` is not needed anymore b/c the `.env` file is loaded automatically.

**Comment by mrutkows** [comment]
@ddragosd Hey Dragos, just looking at devtools PR backlog... and saw this one from May.  Is this still something that needs to be done?  If so, can you think of a good reviewer to verify/approve?

**Comment by rabbah** [comment]
@ddragosd do you want to resolve the conflicts on this?

**Comment by rabbah** [comment]
Closing as stale/abandoned.

**Pull request closed without merging**